### PR TITLE
Support `$comment` query operator

### LINKED ISSUE #168: Support `$comment` query operator
Put `$comment`'s content into generated SQL query after the first statement. For example, `db.collection.find({$comment: "test"})` should generate `SELECT /* test */ * FROM ...`. See https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-COMMENTS.

To prevent SQL injection, we should replace occurrences of `/*` and `*/` in the comment with `/ *` and `* /` (add extra space).

### PR DESCRIPTION
**Pull request by ribaraka**
Closes #168.

### CODE REVIEW

File: internal/handlers/pg/fetch.go
Comment (seeforschauer) [inline internal/handlers/pg/fetch.go:77]: could you please share with us your thought on why the function name is like this?

File: internal/handlers/pg/fetch.go
Comment (seeforschauer) [inline internal/handlers/pg/fetch.go:81]: why not to put the row above inline in the `Sprintf`?

File: internal/handlers/pg/msg_find.go
Comment (seeforschauer) [inline internal/handlers/pg/msg_find.go:98]: you've created a new `fetch` function abandoning the old one. Is there any reason for this?

**Review by seeforschauer — CHANGES_REQUESTED** [review]

File: internal/handlers/pg/fetch.go
Comment (AlekSi) [reply internal/handlers/pg/fetch.go:80]: From ticket description:

> To prevent SQL injection, we should replace occurrences of /* and */ in the comment with / * and * / (add extra space).

See also [there](https://xkcd.com/327/)

File: internal/handlers/pg/msg_find.go
Comment (ribaraka) [reply internal/handlers/pg/msg_find.go:98]: It has been added more to catch a comment at all.

File: internal/handlers/pg/fetch.go
Comment (ribaraka) [reply internal/handlers/pg/fetch.go:81]: it is a good suggestion :)
but in this case I'm trying to avoid long lines and just explicitly expressed the operation (line 80)

File: internal/handlers/pg/msg_find.go
Comment (ribaraka) [reply internal/handlers/pg/msg_find.go:98]: a comment parameter appeared and I created a new prototype function as a suggestion to replace the old one.. I should have mentioned it in the description of the PR. My bad, sorry

File: internal/handlers/pg/fetch.go
Comment (seeforschauer) [reply internal/handlers/pg/fetch.go:85]: yeah it's more visible though

File: internal/handlers/pg/fetch.go
Comment (seeforschauer) [reply internal/handlers/pg/fetch.go:78]: Hi, `protoFetch` is used in msg find and in tests and nowhere else.  
Why not rename `fetch` and add all code to this?

Comment (ribaraka) [reply internal/handlers/pg/fetch.go:78]: actually I'm going to leave `fetch` func.  `protoFetch` is a temporarily prototype as a future view of `fetch` func. I'm not sure if you all okay with this new implementation so I just replaced this in one in one place (for simplicity) and waiting for you approval and command to apply it in every occurrences.

File: internal/handlers/pg/fetch.go
Comment (AlekSi) [inline internal/handlers/pg/fetch.go:111]: Let's add documentation comment and move it before function that uses it

File: internal/handlers/pg/fetch.go
Comment (AlekSi) [inline internal/handlers/pg/fetch.go:86]: I think that will be easier to read without `else`. In pseudocode:
```go
sql := `SELECT `
if param.comment != "" {
  // handle /*, */
  sql += `/* ` + params.comment + ` */ `
}
sql += `_jsonb FROM ` + pgx.Identifier{param.db, param.collection}.Sanitize()
```

**Review by AlekSi — COMMENTED** [review]
`protoFetch` looks good, let's replace existing `fetch` with it

File: internal/handlers/pg/fetch.go
Comment (ribaraka) [reply internal/handlers/pg/fetch.go]: Done.

File: internal/handlers/pg/fetch.go
Comment (ribaraka) [reply internal/handlers/pg/fetch.go]: Done.

**Review by AlekSi — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

**Review by seeforschauer — CHANGES_REQUESTED** [review]
@ribaraka would you please resolve conflicts

File: integration/basic_test.go
Comment (ribaraka) [reply integration/basic_test.go:108]: the block has been refactored and added check for existing schema.

**Review by seeforschauer — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**
